src/features/groups/components/ContainerList/CreateContainerButton/CreateContainerBottomDrawerContent.tsx

```diff
@@ -25,29 +27,35 @@ import { z } from 'zod';
 
 interface ICreateContainerDrawerContentProps {
   /**
-   * boolean, if true, the state of drawer is open
+   * Boolean, if true, the state of drawer is open
    */
   isOpen: boolean;
   /**
-   * function to change the state of the drawer to close
+   * Function to change the state of the drawer to close
    */
   onClose: () => void;
+  /**
+   * An identifier of a group
+   */
+  groupId: IGroup['id'];
 }
 
 /**
- * the content of the creating a new container drawer including form input
- * @param  isOpen boolean, if true, the state of drawer is open
- * @param  onClose function to change the state of the drawer to close
+ * The content of the creating a new container drawer including form input
```

Comment:
I am sorry but this description is little difficult to understand

Reply:
@nick-y-ito OK! I deleted whole documentation here, as no docs in other components. sorry for confusion.